juniper_official/Linecard/aft-server-entry-stats.rule

```diff
@@ -0,0 +1,41 @@
+healthbot {
```

Comment:
Add comments and detail the rule functionality and input variables if any. you can refer any of the factory default rule for e.g. https://github.com/Juniper/healthbot-rules/blob/master/juniper_official/Interfaces/interface-flaps.rule

Reply:
Done